src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -757,6 +757,30 @@ public static string GetLongDescriptionAnnotation(this IEdmModel model, IEdmVoca
             return null;
         }
 
+        /// <summary>
+        /// Gets the collection of qualified type name for term Org.OData.Validation.V1.DerivedTypeConstraint from a target annotatable.
+        /// </summary>
+        /// <param name="model">The model referenced to.</param>
+        /// <param name="target">The target annotatable to find annotation.</param>
+        /// <returns>Null or a collection string of qualifed type name.</returns>
+        public static IList<string> GetDerivedTypeConstraints(this IEdmModel model, IEdmVocabularyAnnotatable target)

```

Comment:
>IList [](start = 22, length = 5)

Should this be an IEnumerable<string>, rather than an IList<string>?  We don't expect the caller to add/remove from the list.

Reply:
Yes. # Fixed.